G3Common/src/users/CeoBiteMe.java

```diff
@@ -10,27 +10,32 @@
 public class CeoBiteMe extends BranchManager{
 
 	/**
-	 * This is the constructor of the class 
+	 * This is the constructor for the CEO Bite Me class
 	 * 
 	 * @param userId
-	 * @param password
 	 * @param isConfirmedInSystem
 	 * @param userFirstName
 	 * @param userLastName
 	 * @param homeBranch
 	 * @param isLoggedIn
-	 * @param idNumber
+	 * @param privateW4cCodeNumber
 	 * @param userEmail
 	 * @param phoneNumber
 	 * @param branchName
 	 */
-	public CeoBiteMe(String userId, String password, boolean isConfirmedInSystem, String userFirstName,
-			String userLastName, Branch homeBranch, boolean isLoggedIn, int idNumber, String userEmail,
-			String phoneNumber, Branch branchName) {
-		super(userId, password, isConfirmedInSystem, userFirstName, userLastName, homeBranch, isLoggedIn, idNumber, userEmail,
+	public CeoBiteMe(String userId, boolean isConfirmedInSystem, String userFirstName, String userLastName,
+			Branch homeBranch, boolean isLoggedIn, int privateW4cCodeNumber, String userEmail, String phoneNumber,
+			Branch branchName) {
+		super(userId, isConfirmedInSystem, userFirstName, userLastName, homeBranch, isLoggedIn, privateW4cCodeNumber, userEmail,
 				phoneNumber, branchName);
 		
 	}
 
-	
+	/**
+	 * This is the toString for this class
+	 */
+	@Override
+	public String toString() {
+		return "CeoBiteMe: " + userFirstName + " " + userLastName + " CEO ID: " + userId;
```

Comment:
same, if you want to keep the userId its up to you since you did it for all of them.

Reply:
fixed